service/controller/app/v1/resource/chartoperator/create.go

```diff
@@ -23,16 +23,22 @@ func (r Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
 		return microerror.Mask(err)
 	}
 
+	// Resource is used to bootstrap chart-operator in tenant clusters.
+	// So for other apps we can skip this step.
+	if key.AppName(cr) != key.ChartOperatorAppName {
+		r.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("no need to install chart-operator for %#q", key.AppName(cr)))
+		r.logger.LogCtx(ctx, "level", "debug", "message", "canceling resource")
+		return nil
+	}
+
```

Comment:
Aligning the error handling in the `tcnamespace`, `tiller` and `chartoperator` resources.

Reply:
Is this the first resource to be handled? then okay.

Reply:
Order of the resources is here. These 3 are special because they only execute for chart-operator app CRs to do the bootstrap process.

With Helm 3 we will be able to get rid of the tiller resource.

https://github.com/giantswarm/app-operator/blob/4a2c42ac1ab0a06d0ab3dbd3267e9a07fda4106e/service/controller/app/v1/resource_set.go#L233-L247